packages/tenderly-hardhat/src/TenderlyNetwork.ts

```diff
@@ -39,8 +41,13 @@ export class TenderlyNetwork {
     this.host = this.tenderlyJsonRpc.defaults.baseURL!;
 
     if (hre.network.name === "tenderly" && "url" in hre.network.config && hre.network.config.url !== undefined) {
-      this.forkID = hre.network.config.url.split("/").pop();
-      logger.info("Fork ID is:", this.forkID);
+      if(hre.network.config.url.includes("devnet")) {
```

Comment:
```suggestion
      if(hre.network.config.url.includes("/devnet/")) {
```
I guess this would be more safe right?